Week-2/Html Basics/htmlTables.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML Tables</title>
+</head>
+<body>
+    <table border="1">
+        <tr>
+            <td colspan="3"><h1>1</h1></td>
+        </tr>
+        <tr>
+            <td colspan="1" rowspan="2"><h1>2</h1></td>
+            <td><h1>3</h1></td>
```

Comment:
The use of `<h1>` elements within table cells is semantically incorrect.` <h1> `is typically used for main page headings, not for table cell content. It's essential to use HTML elements in a way that reflects their semantic meaning.